hw/ip/csrng/doc/checklist.md

```diff
@@ -16,15 +16,15 @@ All checklist items refer to the content in the [Checklist.]({{< relref "/doc/pr
 
 Type          | Item                           | Resolution  | Note/Collaterals
 --------------|--------------------------------|-------------|------------------
-Documentation | [SPEC_COMPLETE][]              | Not Started | [CSRNG Design Spec]({{<relref "hw/ip/csrng/doc" >}})
-Documentation | [CSR_DEFINED][]                | Not Started |
-RTL           | [CLKRST_CONNECTED][]           | Not Started |
-RTL           | [IP_TOP][]                     | Not Started |
-RTL           | [IP_INSTANTIABLE][]            | Not Started |
-RTL           | [PHYSICAL_MACROS_DEFINED_80][] | Not Started |
-RTL           | [FUNC_IMPLEMENTED][]           | Not Started |
-RTL           | [ASSERT_KNOWN_ADDED][]         | Not Started |
-Code Quality  | [LINT_SETUP][]                 | Not Started |
+Documentation | [SPEC_COMPLETE][]              | Done        | [CSRNG Design Spec]({{<relref "hw/ip/csrng/doc" >}})
+Documentation | [CSR_DEFINED][]                | Done        |
```

Comment:
@cdgori @moidx @tjaychen You probably reviewed the CSRNG spec most: Do you feel that the CSRNG spec is in the following state, as required by https://docs.opentitan.org/doc/project/checklist/index.html for D1:

> Specification mostly (90%) complete, all features are defined.

Reply:
yes i think so.
The big remaining items aren't something we can conclude at the moment until labs give their input.